src/utils/template.js

```diff
@@ -51,15 +51,15 @@ function fillDashboardTemplate(viewedItems) {
   // Summary toggle
   let summaryPageToggle = document.querySelector(summaryPageToggleSelector);
   summaryPageToggle.textContent = chrome.i18n.getMessage('summary');
-  let summaryPageToggleIcon = createIconNode('list');
-  summaryPageToggle.prepend(summaryPageToggleIcon);
+  //let summaryPageToggleIcon = createIconNode('list');
```

Comment:
It's definitely better to do this with CSS, thanks!. 
This commented lines of code will be removed before merging in the master branch

Reply:
Perfect! thanks!